src/braket/ir/jaqcd/program_v1.py

```diff
@@ -216,6 +227,8 @@ def validate_instructions(cls, value, field):
                 return _valid_gates[value["type"]](**value)
             elif value["type"] in _valid_noise_channels:
                 return _valid_noise_channels[value["type"]](**value)
+            elif value["type"] in _valid_compiler_directives:
+                return _valid_compiler_directives[value["type"]](**value)
```

Comment:
Annoying that the coverage still isn't getting there.

Reply:
Updated tests to get 100% coverage.